src/transport/AdminPairingTable.h

```diff
@@ -189,14 +189,14 @@ class DLL_EXPORT AdminPairingInfo
         uint64_t mFabricId; /* This field is serialized in LittleEndian byte order */
         uint16_t mVendorId; /* This field is serialized in LittleEndian byte order */
 
-        char mFabricLabel[kFabricLabelMaxLengthInBytes + 1] = { '\0' };
-
         uint16_t mRootCertLen; /* This field is serialized in LittleEndian byte order */
         uint16_t mOpCertLen;   /* This field is serialized in LittleEndian byte order */
 
         Crypto::P256SerializedKeypair mOperationalKey;
         uint8_t mRootCert[kMaxChipCertSize];
         uint8_t mOperationalCert[kMaxChipCertSize];
+
+        char mFabricLabel[kFabricLabelMaxLengthInBytes + 1] = { '\0' };
```

Comment:
Whatever required this change in the member order, please file an issue on it?  The order here should not matter as long as we are not trying to read stale data from somewhere... or something is going badly wrong.  In any case, we need to diagnose what the problem is here and fix it correctly.